forgerock-auth/src/main/java/org/forgerock/android/auth/Config.java

```diff
@@ -37,6 +37,9 @@ public class Config {
     private String redirectUri;
     private String scope;
     private String oAuthUrl;
+    private Long oauthCacheMillis;
```

Comment:
By default, those values are 0 which means no caching, however, those values are used in the previous version of SDK already, the encrypt and encrypt process are heavy, it may be good to have this option. We may need to discuss this feature alignment between platforms.

Reply:
so we are keeping it ?